flocker/provision/_install.py

```diff
@@ -96,6 +96,14 @@ def run_with_fabric(username, address, commands):
 run = run_with_fabric
 
 
+def task_test_homebrew(recipe_url='flocker'):
```

Comment:
:question: Is this default necessary? Without `brew tap clusterhq/flocker` it won't work, and even then it is just happenstance that it would work (the `flocker` recipe should be deleted).